packages/@dcl/playground-assets/etc/playground-assets.api.md

```diff
@@ -1544,6 +1549,9 @@ export interface MapComponentDefinition<T> extends LastWriteWinElementSetCompone
     createOrReplace(entity: Entity, val?: Partial<T>): T;
 }
 
+// @public (undocumented)
+export const MapPin: LastWriteWinElementSetComponentDefinition<PBMapPin>;

```

Comment:
I'm not sure why a few MapPin related snippets appeared when I built protocol, maybe they were missing? I'm not sure if I should remove them from this PR

@leanmendoza @gonpombo8 ?

Reply:
https://github.com/decentraland/js-sdk-toolchain/blob/main/scripts/protocol-buffer-generation/index.ts#L13
add here the component id of that MapPin component

Reply:
Done!